bookwyrm/views/preferences/export.py

```diff
@@ -92,7 +103,7 @@ def post(self, request):
         )
 
 
-# pylint: disable=no-self-use
+# pylint: disable=no-self-use,too-many-locals
```

Comment:
This should be above `class Export(View)` rather than class `ExportUser(View)`

Reply:
Thank you, I have literally no idea of what I'm doing here. I do not understand the "too-many locals" error nor the solution.

Reply:
Essentially there's nothing wrong with your code, but pylint flags it as a potential problem: "Some programmers consider using several local variables in a function or method is an indicator that the function or method is too complex, or trying to do too much."

What we're doing by adding this `disable` comment at the top is saying "Thanks for the tip, Pylint, don't warn me about this problem here". It's a judgement call: I'm judging we can ignore pylint in this case.